### PR TITLE
Add version: ShiningLight.OpenSSL.Light version 1.1.1v

### PR DESCRIPTION
**Pull request by szhu25**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/114624&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/ShiningLight/OpenSSL/Light/1.1.1v/ShiningLight.OpenSSL.Light.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL_Light-1_1_1v.msi
+  InstallerSha256: 2B4AC9DFC592EB3BCEE9CA01A0DBF3CB30B4C48B275900E5023FBBA3B99AEDDE
+  ProductCode: '{39C5F6AE-255C-4AAF-A083-1AEC152A5AE1}'
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://slproweb.com/download/Win64OpenSSL_Light-1_1_1v.exe
+  InstallerSha256: D9433700C93855BD3090E132EE29C0A8159F7BA92FDF5F8B0BCFC5E3B7D8CED5
+  ProductCode: OpenSSL Light (64-bit)_is1
+  AppsAndFeaturesEntries:
+  - DisplayName: OpenSSL 3.0.8 Light (64-bit)
```
Comment (ItzLevvie) [inline manifests/s/ShiningLight/OpenSSL/Light/1.1.1v/ShiningLight.OpenSSL.Light.installer.yaml:34]:
```suggestion
  - DisplayName: OpenSSL 1.1.1v Light (64-bit)
```

File: manifests/s/ShiningLight/OpenSSL/Light/1.1.1v/ShiningLight.OpenSSL.Light.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    Publisher: OpenSSL Win64 Installer Team
+    ProductCode: OpenSSL Light (64-bit)_is1
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL_Light-1_1_1v.exe
+  InstallerSha256: A451BFEFF5041B6232AD66B909249A5124412BDB8B493A90ABFAA1D43D717CF0
+  ProductCode: OpenSSL Light (32-bit)_is1
+  AppsAndFeaturesEntries:
+  - DisplayName: OpenSSL 3.0.8 Light (32-bit)
```
Comment (ItzLevvie) [inline manifests/s/ShiningLight/OpenSSL/Light/1.1.1v/ShiningLight.OpenSSL.Light.installer.yaml:43]:
```suggestion
  - DisplayName: OpenSSL 1.1.1v Light (32-bit)
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by szhu25** [comment]
Hi @trenly, could you help me run the Azure pipeline checks for these OpenSSL updates? There are 6 PR for the 3 updated OpenSSL version (full and light).

Thank you!

**Pull request merged**